.circleci/config.yml

```diff
@@ -1,8 +1,111 @@
-version: 2
+version: 2.0
+
+anchor_test_env: &anchor_test_env
+  docker:
+    - image: googleapis/artman:0.14.3
```

Comment:
Will we need to manually bump this when we get new releases?

Reply:
Updated in go/artman-release-instructions.